lib/collector/remote-method.js

```diff
@@ -74,6 +78,16 @@ RemoteMethod.prototype.serialize = function serialize(payload, callback) {
  * @param {Function} callback          What to do next. Gets passed any error.
  */
 RemoteMethod.prototype.invoke = function invoke(payload, nrHeaders, callback) {
+  const reportDataUsage = (sent, received) => {
```

Comment:
nice consolidation of this logic. However, this is creating a function every time invoke is getting called. I'm sure it will be optimized by v8 but you could just put on the instance of RemoteMethod as `this_metrics` will be there right?

Reply:
good point. i'll refactor accordingly soon

Reply:
Did that refactor, does it look good?

Reply:
Sure does!